test/e2e/azure_test.go

```diff
@@ -499,9 +501,13 @@ var _ = Describe("Workload cluster creation", func() {
 		})
 	})
 
+	// ci-e2e.sh and Prow CI skip this test by default. To include this test, set `GINKGO_SKIP=""`.
+	// This spec expects a user-assigned identity named "cloud-provider-user-identity" in a "capz-ci"
+	// resource group. Override these defaults by setting the IDENTITY_NAME and CI_RG environment variables.
```

Comment:
we're actually getting rid of this test in #2292, is there another in-tree optional test we can use to add user assigned id coverage?

Reply:
Dual-stack, or GPU. I vote for dual-stack.